src/edu/csus/ecs/pc2/ui/ShadowCompareRunsPane.java

```diff
@@ -126,12 +126,11 @@ public String getPluginTitle() {
     public ShadowCompareRunsPane(ShadowController shadowController) {
         Dimension size = new Dimension(700,700);
         this.setPreferredSize(size);
-        this.setMinimumSize(size);
-        
+        this.setMinimumSize(size);shadowController.setFilter(FILTERS.NONE);//Required to reset the state of the filter
```

Comment:
Break this up into 2 lines.  We don't like to put multiple statements on a single line.